hub/src/main/java/com/hub/HubController.java

```diff
@@ -95,20 +94,16 @@ protected Long addServer(@RequestBody String json) throws Exception {
         return serverId;
     }
 
-    private void sendLeader() throws IOException {
+    private void sendLeader() throws Exception {
```

Comment:
sending the leader might benefit from a retry

Reply:
Anytime a server starts back up it calls getLeader so the hub shouldn't have to retry sending the leader

Reply:
good point